ui-meta/sample-appliance.yml

```diff
@@ -6,51 +7,52 @@ label: "Sample Appliance"
+# Alternatively, as shown here, a data URL can be used to encode the image directly
+logo:
```

Comment:
I think this will break - the linter has made an assuption about `data:` being a key and dropped it onto a newline.

We should probably restore what was there and stringify it by wrapping it in `"`.